web/config.py

```diff
@@ -842,6 +842,8 @@
 ##########################################################################
 SHARED_STORAGE = []
 
+DISABLED_LOCAL_PASSWORD_STORAGE = False
```

Comment:
Since DISABLED_LOCAL_PASSWORD_STORAGE is not user configurable and used internally, it should be removed from config.py. Use it internally or put it in evaluate_config.

Reply:
> Since DISABLED_LOCAL_PASSWORD_STORAGE is not user configurable and used internally, it should be removed from config.py. Use it internally or put it in evaluate_config.

Done, moved DISABLED_LOCAL_PASSWORD_STORAGE to evaludate_config